internal/eval/store_eval.go

```diff
@@ -453,69 +453,6 @@ func evalZRANGE(args []string, store *dstore.Store) *EvalResponse {
 	}
 }
 
-// evalZRANK returns the rank of the member in the sorted set stored at key.
```

Comment:
Hi @sashpawar11 I see this is mistakenly got removed.

Reply:
Gosh, my bad - might have missed it during conflict resolution. Will fix and add it back soon